src/redux/sagas/motions/expenditures/editLockedExpenditureMotion.ts

```diff
@@ -0,0 +1,252 @@
+import {
+  type AnyVotingReputationClient,
+  ClientType,
+  ColonyRole,
+  getPermissionProofs,
+  Id,
+} from '@colony/colony-js';
+import { BigNumber } from 'ethers';
+import { call, fork, put, takeEvery } from 'redux-saga/effects';
+
+import { ADDRESS_ZERO, APP_URL } from '~constants';
+import { ActionTypes } from '~redux/actionTypes.ts';
+import {
+  createGroupTransaction,
+  createTransaction,
+  createTransactionChannels,
+  waitForTxResult,
+} from '~redux/sagas/transactions/index.ts';
+import {
+  getColonyManager,
+  initiateTransaction,
+  getMulticallDataForUpdatedPayouts,
+  uploadAnnotation,
+  getPayoutsWithSlotIds,
+} from '~redux/sagas/utils/index.ts';
+import { type Action } from '~redux/types/index.ts';
+import { type ExpenditurePayoutFieldValue } from '~types/expenditures.ts';
+import { takeFrom } from '~utils/saga/effects.ts';
+
+export type EditExpenditureMotionPayload =
+  Action<ActionTypes.MOTION_EDIT_LOCKED_EXPENDITURE>['payload'];
+
+function* editLockedExpenditureMotion({
+  payload: {
+    colonyAddress,
+    expenditure,
+    payouts,
+    networkInverseFee,
+    annotationMessage,
+    motionDomainId,
+  },
+  meta,
+  meta: { setTxHash, id },
+}: Action<ActionTypes.MOTION_EDIT_LOCKED_EXPENDITURE>) {
+  const colonyManager = yield getColonyManager();
+  const colonyClient = yield colonyManager.getClient(
+    ClientType.ColonyClient,
+    colonyAddress,
+  );
+
+  const batchId = 'motion-edit-locked-expenditure';
```

Comment:
Could you align this saga with the fixes I did in #2020 (e.g. https://github.com/JoinColony/colonyCDapp/pull/2020/files#diff-5097462c62b8afbf61c34f5ba821182563002e86f110568703f0e0b762948b4d) - especially around the batch ID and `createGroupTransaction` params?